mobile-widgets/qmlmanager.cpp

```diff
@@ -315,14 +315,14 @@ void QMLManager::openLocalThenRemote(QString url)
 		auto credStatus = qPrefCloudStorage::cloud_verification_status();
 		if (credStatus != qPrefCloudStorage::CS_NOCLOUD &&
 		    credStatus != qPrefCloudStorage::CS_INCORRECT_USER_PASSWD)
-			QMLPrefs::instance()->setCredentialStatus(qPrefCloudStorage::CS_NEED_TO_VERIFY);
+			qPrefCloudStorage::set_cloud_verification_status(qPrefCloudStorage::CS_NEED_TO_VERIFY);
```

Comment:
`qmlprefs::setCredentialStatus() only makes something special in case of CS_NOCLOUD.`
I'm sure that made sense to you when you wrote it. But to the reader that's at best a riddle.
The commit message only confuses me - so you are obviously replacing the `setCredentialStatus()` calls with `set_cloud_verification_status()`. Can you describe why that's the right thing to do? And if there is a special case that is worth explaining (`makes something special`), then maybe be more explicit about that?

Reply:
refrased